### PR TITLE
7.3 What's New

### PR DESCRIPTION
**Pull request by Pessimistress**
For #3591

### CODE REVIEW

File: docs/whats-new.md [hunk 1]
```diff
@@ -4,26 +4,42 @@ This page contains highlights of each deck.gl release. Also check our [vis.gl bl
...
 
 ### Tile3DLayer
 
-deck.gl has partnered with [Cesium](https://cesium.com/) to implement support for the [3D Tiles specification](https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/specification). This makes it possible to dynamically load and render infinitely large datasets into the browser.
+deck.gl has partnered with [Cesium](https://cesium.com/) to implement support for the OGC [3D Tiles specification](https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/specification). This makes it possible to dynamically load and render infinitely large datasets into the browser.
 
 In this initial release, the layer has full support for point clouds and experimental support for glTF tiles. Try the [demo]() for yourself.
 
-### Built-in Tooltip
+### Python Binding and Jupyter Widget
```
Comment (ibgreen) [inline docs/whats-new.md]: "Python Binding and Jupyter Widget" => 

"pydeck: deck.gl for Python and Jupyter Notebooks"

File: docs/whats-new.md [hunk 2]
```diff
@@ -4,26 +4,42 @@ This page contains highlights of each deck.gl release. Also check our [vis.gl bl
...
 
-A new prop [getTooltip](/docs/api-reference/deck.md#gettooltip-function-optional) is added to the `Deck` class (and inheritly the `DeckGL` react component). By supplying this callback, an app may specify the content and styling of a built-in tooltip. This takes away the hassle of creating and maintaining your own DOM element.
+We have released a python module [pydeck]() for Python developers to interact with deck.gl via a native API. A Jupyter Notebook widget is also available to visualize your data right in your notebook. See [documentation]() and our release [blog post]() for details.
```
Comment (ibgreen) [inline docs/whats-new.md]: - Python is capitalized...
- Clarify by mentioning `pip install`: `python module [pydeck]()` => `Python module [pip install pydeck](..)`  ?  
- Clearer: `via a native API` => `via a native Python API`.
- Do data scientists know about "widgets"?: `A Jupyter Notebook widget is also available` => pydeck is also integrated with Jupyter Notebook, enabling you to interactively create deck.gl visualizations right in your notebooks.

@ajduberstein

File: docs/whats-new.md [hunk 3]
```diff
@@ -4,26 +4,42 @@ This page contains highlights of each deck.gl release. Also check our [vis.gl bl
...
 
-### Python Binding and Jupyter Widget
+### Transitions
```
Comment (ibgreen) [inline docs/whats-new.md]: Transitions => Transition System Improvements

File: docs/whats-new.md [hunk 4]
```diff
@@ -40,13 +56,15 @@ For custom layer authors: the attribute system is simplified. One may now use `t
 ### Minor Improvements and Bug Fixes
```
Comment (ibgreen) [inline docs/whats-new.md]: "Minor Improvements and Bug Fixes" => "Other Features" / "Other Improvements"

File: docs/whats-new.md [hunk 5]
```diff
@@ -40,13 +56,15 @@ For custom layer authors: the attribute system is simplified. One may now use `t
...
 
 - `OrbitController` now supports 360 degree rotation on both axis. Relax `minRotationX` and `maxRotationX` to use this feature.
+- Built-in tooltip: A new prop [getTooltip](/docs/api-reference/deck.md#gettooltip-function-optional) is added to the `Deck` class. By supplying this callback, an app may specify the content and styling of a built-in tooltip.
```
Comment (ibgreen) [inline docs/whats-new.md]: Auto-tooltip seems like a new feature and worth its own entry at the end of the list above.

**Comment by 1chandu** [comment]
Can you rebase before merging, I added an entry for `WebGL1 support for HeatmapLayer` to whats-new doc.

**Pull request merged**